tests/Integration/Behaviour/Features/Scenario/Product/add_to_pack.feature

```diff
@@ -0,0 +1,39 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags add-to-pack
+@reset-database-before-feature
+@clear-cache-after-feature
+@add-to-pack
+Feature: Add product to pack from Back Office (BO)
+  As a BO user
+  I need to be able to add product to pack from BO
+
+  Scenario: I add standard product to a pack
+    Given I add product "productPack1" with following information:
+      | name       | en-US: weird sunglasses box |
+      | is_virtual | false                       |
+    And product "productPack1" type should be standard
+    And I add product "product2" with following information:
+      | name       | en-US: shady sunglasses     |
+      | is_virtual | false                       |
+    And product "product2" type should be standard
+    When I pack 5 standard product "product2" to a pack of "productPack1"
```

Comment:
This is not easy to understand I would suggest:
```suggestion
    When I add 5 products "product2" to a pack of "productPack1"
```

(also remove the type as suggested above because it's confusing especially you add virtual products.. ^^)

Reply:
okay.
I thought we'll need different string for combinational product, but now I think that we can use the same string as you suggest, without mentioned type and we will get the combination id from the reference.. So later ill need to refactor to save product_id and combination_id inside product reference instead of id only.